source/euler/hyperbolic_system.h

```diff
@@ -1313,5 +1339,159 @@ namespace ryujin
       return result;
     }
 
+
+    template<int dim, typename Number>
+    void HyperbolicSystem::View<dim, Number>::post_process(const vector_type &U,
+                                                           Number t,
+                                                           MPI_Comm mpi_communicator_,
+                                                           const ryujin::OfflineData<dim, Number>& offline_data_)
+    const
+    {
+    if(dim == 2){
+      using scalar_type = dealii::LinearAlgebra::distributed::Vector<Number>;
+
+
+       //some general variables
+       dealii::Point<dim> disk_center;//center at 0,0 for every case, how to not hardcode this?
+       //FIXME: how do we calculate disk center from the information in mesh generation?
+       double disk_radius = 0.25;
+
+       //first, set up the finite element, the data, and the facevalues
+       dealii::FE_Q<dim> fe(ORDER_FINITE_ELEMENT);//the finite element
```

Comment:
This is an expensive operation. Isn't there a finite element object sitting around somewhere?